javascriptv3/example_code/bedrock-runtime/scenarios/converse_tool_scenario/converse-tool-scenario.js

```diff
@@ -0,0 +1,314 @@
+async function callWeatherTool(longitude, latitude) {
+  // Open-Meteo API endpoint
+  const apiUrl = `https://api.open-meteo.com/v1/forecast?latitude=${latitude}&longitude=${longitude}&current_weather=true`;
+
+  // Fetch the weather data.
+  return fetch(apiUrl)
+    .then((response) => {
+      return response.json().then((current_weather) => {
+        return current_weather;
+      });
+    })
+    .catch((error) => {
+      console.error("Error fetching weather data:", error);
+    });
+}
+/**
+ * Used repeatedly to have the user press enter.
+ * @type {ScenarioInput}
+ */
+const pressEnter = new ScenarioInput("continue", "Press Enter to continue", {
+  type: "confirm",
+});
+
+const greet = new ScenarioOutput(
+  "greet",
+  "Welcome to the Amazon Bedrock Tool Use demo! \n" +
+    "This assistant provides current weather information for user-specified locations. " +
+    "You can ask for weather details by providing the location name or coordinates." +
+    "Weather information will be provided using a custom Tool and open-meteo API." +
+    "For the purposes of this example, we'll use in order the questions in ./questions.json :\n" +
+    "What's the weather like in Seattle? " +
+    "What's the best kind of cat? " +
+    "Where is the warmest city in Washington State right now? " +
+    "What's the warmest city in California right now?\n" +
+    "To exit the program, simply type 'x' and press Enter.\n" +
+    "Have fun and experiment with the app by editing the questions in ./questions.json! " +
+    "P.S.: You're not limited to single locations, or even to using English! ",
+
+  { header: true },
+);
+const displayAskQuestion1 = new ScenarioOutput(
+  "displayAskQuestion1",
+  "Press enter to ask question number 1 (default is 'What's the weather like in Seattle?')",
+);
+
+const askQuestion1 = new ScenarioAction(
+  "askQuestion1",
+  async (/** @type {State} */ state) => {
+    const userMessage1 = data.questions["question-1"];
+    await askQuestion(userMessage1);
+  },
+);
+
+const displayAskQuestion2 = new ScenarioOutput(
+  "displayAskQuestion2",
+  "Press enter to ask question number 2 (default is 'What's the best kind of cat?')",
+);
+
+const askQuestion2 = new ScenarioAction(
+  "askQuestion2",
+  async (/** @type {State} */ state) => {
+    const userMessage2 = data.questions["question-2"];
+    await askQuestion(userMessage2);
+  },
+);
+const displayAskQuestion3 = new ScenarioOutput(
+  "displayAskQuestion3",
+  "Press enter to ask question number 3 (default is 'Where is the warmest city in Washington State right now?')",
+);
+
+const askQuestion3 = new ScenarioAction(
+  "askQuestion3",
+  async (/** @type {State} */ state) => {
+    const userMessage3 = data.questions["question-3"];
+    await askQuestion(userMessage3);
+  },
+);
+
+const displayAskQuestion4 = new ScenarioOutput(
+  "displayAskQuestion4",
+  "Press enter to ask question number 4 (default is 'What's the warmest city in California right now?')",
```

Comment:
The questions all say "y/n" but only ask me to press Enter. Is there a way we can remove the y/n?